eth/main.cpp

```diff
@@ -1108,10 +1108,11 @@ int main(int argc, char** argv)
 			}
 		}
 
-		while (web3.ethereum()->blockQueue().items().first + web3.ethereum()->blockQueue().items().second > 0)
+		bool moreToImport = true;
+		while (moreToImport)
 		{
 			this_thread::sleep_for(chrono::seconds(1));
-			web3.ethereum()->syncQueue(100000);
+			tie(ignore, moreToImport, ignore) = web3.ethereum()->syncQueue(100000);
```

Comment:
It is basically the same as checking only `web3.ethereum()->blockQueue().items().first > 0`